### ISSUE TITLE
Push v1.1.2 to npm

### ISSUE THREAD

**Issue by ath88** [mesthabro/react-gradient-progress#19]
Latest on npm is v1.0.3

Any chance the newest version could be pushed?

**Comment by MarioSerano** [comment]
I agree, it's not updating with React 17 and thus making us use npm install --legacy-peer-deps...
